neural_compressor/model/torch_model.py

```diff
@@ -558,7 +558,7 @@ def export_compressed_model(
                 )
                 new_module.pack(int_weight, gptq_scale, gptq_zp, m.bias, gptq_perm)
                 set_module(self.model, k, new_module)
-        elif autoround_config:
+        elif autoround_config and (device == "cpu" or device == "auto"):
```

Comment:
does ITREX support inference with NV GPU?  if support,  does it go through device == "auto" path?

Reply:
we support NV GPU if cuda available, when device is "auto", it will detect cuda avaliable first and then use cpu device.